src/services/sync/strategies/TSP/TSPSchoolSyncer.js

```diff
@@ -291,8 +295,8 @@ class TSPSchoolSyncer extends mix(Syncer).with(ClassImporter) {
 			this.stats.errors.push({
 				type: 'update-teacher',
 				entity: tspTeacher.lehrerUid,
-				message: `Lehrer "${tspTeacher.lehrerVorname} ${tspTeacher.lehrerNachname}"`
-					+ ' konnte nicht aktualisiert werden.',
+				message:
+					`Lehrer "${tspTeacher.lehrerVorname} ${tspTeacher.lehrerNachname}"` + ' konnte nicht aktualisiert werden.',
```

Comment:
Codacy found an issue: [Unexpected string concatenation of literals.](https://app.codacy.com/gh/hpi-schul-cloud/schulcloud-server/pullRequest?prid=6074601)